src/features/quiz/SymbiosisQuiz.js

```diff
@@ -0,0 +1,138 @@
+/* eslint-disable */
+
+import React, { useState } from "react";
+import Progress from "./Progress";
+import Questions from "./questions/Questions";
+import Answers from "./answers/Answers";
+import { makeStyles } from "@material-ui/core/styles";
+import { Box, Typography, Button } from "@material-ui/core";
+
+const useStyles = makeStyles({
+  Container: {
+    width: "100%",
+    height: "70vh",
+    display: "flex",
+    flexDirection: "column",
+    justifyContent: "center",
+    alignItems: "center",
+  },
+});
+
+export default function SymbiosisQuiz() {
+  const classes = useStyles();
+  const [currentQuestion, setCurrentQuestion] = useState(0);
+  const [currentAnswer, setCurrentAnswer] = useState("");
+  const [answers, setAnswers] = useState([]);
+  const [showResults, setShowResults] = useState(false);
+  const [error, setError] = useState([]);
+
+  const questions = [
+    {
+      id: 1,
+      question:
+        "On April 16th which US states contain the top 2 COVID - 19 hot spots?",
```

Comment:
Again, not to butt in but to save time, I believe this question should be
"Which animal has 3 relationships with other animals, including both mutualism and commensalism?"

Reply:
This is what happens when I copy and paste too much lol